src/configAPIClient.ts

```diff
@@ -14,30 +14,33 @@ export type SDKCompleteInitCallback = (
     mpInstance: MParticleWebSDK
 ) => void;
 
-export interface IKitConfigs {
+export interface IKitConfigs extends IKitFilterSettings {
```

Comment:
I split this out because the [filterDictionary](https://github.com/mParticle/mparticle-web-sdk/pull/564/files#diff-a5066c42211de5a8e75e8ee953c4a6038b5bbbc49e353fe1bf034392b121c9fbR10) only contains the `IKitFilterSettings` and not all settings. Another option was to keep this as is, and do an Omit<IKitConfigs, //several items> but that seemed too verbose.